botocore/utils.py

```diff
@@ -364,6 +366,39 @@ def is_global_accesspoint(context):
     return is_global
 
 
+# All code related to experimental plugins is considered a private interface and
+# subject to abrupt breaking changes or removal without a minor version bump
+BOTOCORE_EXPERIMENTAL__PLUGINS_CONTEXT = contextvars.ContextVar(
+    'BOTOCORE_EXPERIMENTAL__PLUGINS', default=None
```

Comment:
Do we want to have this use the value from the environment so we're checking a singular place?

Reply:
This is removed now, so resolving.  The reason that would have been difficult is that it would have been resolved at module import time, before we had the chance to mock out any environment variables.